src/XYPlot.js

```diff
@@ -55,17 +39,14 @@ function getMouseOptions(
   const xValue = !_.inRange(
     innerX,
     0,
-    chartSize.width /* + padding.left + padding.right */
+    chartSize.width + marginLeft + marginRight
```

Comment:
why was this commented out? does this work as expected?

Reply:
Investigating it more to make sure I don't break anything, but basically the margins are added to the chart and should be included as the overall width. Looking into yValue and if marginTop and marginBottom should also be included when thinking of the height.

Reply:
Will remove adding of margins 👍 Tested it out on XYPlot and I think it's fine without adding this to the "range" that a mouse position should be in